src/coreclr/jit/emitriscv64.cpp

```diff
@@ -2427,6 +2440,14 @@ static constexpr unsigned kInstructionFunct7Mask = 0xfe000000;
             assert(rs1 < 32);
             assert((opcode & kInstructionFunct7Mask) == 0);
             break;
+        case INS_fence:
+        {
+            assert(rd == REG_ZERO);
+            assert(rs1 == REG_ZERO);
+            ssize_t format = immediate >> 8;
+            assertCodeLength(format, 3);
```

Comment:
I cannot understand what it means. It looks `assertCodeLength(format, 8)` is ok. Could you please explain?

Reply:
Sorry for this bug, this functions have been synchronized with my latest changes, but the fence usage haven't been tested or used yet hence this bug was not noticed. I've wanted to check whether the format of the fence is equal to 0b0000 or 0b1000 since currently those are only allowed values. It seems that I've too eagerly replaced the `assert((format & 0x7) == 0)` with this line of code, although as I said this function is never called with the fence instruction yet so it wouldn't cause any regression. Thanks for pointing that out

Reply:
Thank you.